e2e/manifests/staging.toml

```diff
@@ -3,6 +3,8 @@ public_chains = ["op_sepolia","base_sepolia","arb_sepolia"]
 anvil_chains = ["slow_l1"]
 multi_omni_evms = true
 prometheus = true
+pinned_relayer_tag = "2a0a9bb"
```

Comment:
We shouldn't pin staging, can just use latest in all cases

Reply:
Yep. Only added that so we could test it. But after that has been verified we should remove it.